corehq/motech/repeaters/tasks.py

```diff
@@ -124,7 +107,8 @@ def check_repeaters_in_partition(partition):
             "commcare.repeaters.check.processing",
             timing_buckets=_check_repeaters_buckets,
         ):
-            for record in _iterate_repeat_records_for_partition(start, partition, CHECK_REPEATERS_PARTITION_COUNT):
+            for record in SQLRepeatRecord.objects.iter_partition(
+                    start, partition, CHECK_REPEATERS_PARTITION_COUNT):
```

Comment:
Probably outside of the scope of this PR, but I'm pretty sure I created the name `CHECK_REPEATERS_PARTITION_COUNT` and I don't like it. In reality it should be `REPEAT_RECORD_PARTITION_COUNT`, or maybe you can think of a better name. No need to change here, but wanted to at least have the conversation about it and flag it.

Reply:
I agree, your suggested alternate name is better. Unfortunately I think this is going to be a slightly complicated change if we want to rename that because it's the name of a setting that is referenced in commcare-cloud and possibly localsettings files out in the wild. I'm not going to change this as part of this Couch to SQL migration.